### ISSUE TITLE
Cannot read property 'version' of null

### ISSUE THREAD

**Issue by enshuti** [Microsoft/PowerBI-visuals-tools#251]
Hi, 

Anybody else getting this error:
```
Cannot read property 'version' of null
```
when running command `pbiviz start`? 
I'm getting even the error even though I've started a fresh visuals project using the current **powerbi-visuals-tools beta**. I have no issue running a 2.x.x project. I followed instructions in [this demo](https://www.youtube.com/watch?v=HRMLdaB3ogs&t=3183s)

**Comment by zBritva** [comment]
Hello @enshuti , which version of beta tools do you use? Is problem exists with tools 3.0.12?

**Comment by enshuti** [comment]
Hello @zBritva It exists in v3.0.11. 
But I found out the error only occures when running running the project using `pbiviz start` command. 
Running `npm run start` instead, seemed to do the trick. 
Not sure why since the script just runs the same `pbiviz start`

**Comment by zBritva** [comment]
`npm run start` runs local version of tools, specified in package.json

**Comment by ScriptPup** [comment]
Bump, I have this same issue except it doesn't work when I run `npm run start` either. I get the same error message.

**Comment by JeyanthVictory** [comment]
Hi,

I'm having the same issue. I just followed the instructions provided in 
https://docs.microsoft.com/en-us/power-bi/developer/custom-visual-develop-tutorial#prerequisites
It throws error.
![image](https://user-images.githubusercontent.com/36911889/65420377-9e6f7300-de1e-11e9-852b-8430481c90bc.png)
Hope I get some solution.
Thanks in advance.

**Comment by zBritva** [comment]
@JeyanthVictory could you specify the version of tools and how you run start command (local version or global version of installed tools)

@ScriptPup Do you have "powerbi-visuals-tools" in package.json in you visual project?

**Comment by JeyanthVictory** [comment]
@zBritva , I have installed the power-bi tools globally.
![image](https://user-images.githubusercontent.com/36911889/65423490-c1058a00-de26-11e9-9563-333d0af0f261.png)

And this's how I did run start

1.`npm i -g powerbi-visuals-tools`
2.`pbiviz --install-cert`
3.Configured certificates.
4.Created a new project `pbiviz new CircleCard` (in custom directory not in C:/)
5.`pbiviz start` after locating to project directory

For your info, this is my package.json

{
  "name": "visual",
  "scripts": {
    "pbiviz": "pbiviz",
    "start": "pbiviz start",
    "package": "pbiviz package",
    "lint": "tslint -r \"node_modules/tslint-microsoft-contrib\"  \"+(src|test)/**/*.ts\""
  },
  "dependencies": {
    "@babel/runtime": "^7.5.5",
    "@babel/runtime-corejs2": "^7.5.5",
    "@types/d3": "5.7.2",
    "core-js": "3.2.1",
    "d3": "5.10.0",
    "powerbi-visuals-api": "^2.6.1",
    "powerbi-visuals-utils-dataviewutils": "^2.2.1"
  },
  "devDependencies": {
    "ts-loader": "6.0.4",
    "typescript": "3.5.3"
  }
}

**Comment by zBritva** [comment]
JeyanthVictory and `pbiviz start` gives to you the error described above? Do you make some changes in the code?

You install the latest version of tools, and if you try to use tutorial from
https://docs.microsoft.com/en-us/power-bi/developer/custom-visual-develop-tutorial#prerequisites

It may not work, because it is outdated. We are updating the tutorial to use with new tools.

**Comment by JeyanthVictory** [comment]
@zBritva , I didn't make any changes in the code. I just created a new project and did `pbiviz start`. If [this ](https://docs.microsoft.com/en-us/power-bi/developer/custom-visual-develop-tutorial#prerequisites)is outdated, can you please suggest the updated one..??

**Comment by JeyanthVictory** [comment]
I just want you to inform that when I run `pbiviz start` , it throws no repository field and no license field.

![image](https://user-images.githubusercontent.com/36911889/65432124-4ba2b500-de38-11e9-9049-722e1d9a7295.png)

Hope this may help you to resolve my issue.

**Comment by JeyanthVictory** [comment]
@helen508light , I followed as you suggested. This's what it throws

![image](https://user-images.githubusercontent.com/36911889/65514393-da780600-defa-11e9-9333-2a82f2b9aa1a.png)

Kindly help me to get rid of this.

**Comment by helen508light** [comment]
Have you specified pbiviz.json file as in example?

**Comment by helen508light** [comment]
Looks strange. Could you share the source code?

**Comment by JeyanthVictory** [comment]
@helen508light , I'm using your code downloaded from https://github.com/Microsoft/PowerBI-visuals-circlecard and changed `pbiviz.json` file from https://github.com/microsoft/powerbi-visuals-gantt/blob/master/pbiviz.json as you said.

For your info,
My pbiviz.json file
`{
  "visual": {
    "name": "circlecard",
    "displayName": "circlecard",
    "guid": "circlecard4AD8D1E1DA0943AEB18E1E59B92001A9",
    "visualClassName": "Visual",
    "version": "1.0.0",
    "description": "A Gantt chart is a type of bar chart which illustrates a project timeline or schedule. The Gantt Chart visual shows the Tasks, Start Dates, Durations, % Complete, and Resources for a project. The Gantt Chart visual can be used to show current schedule status using percent-complete shadings and a vertical \"TODAY\" line. The Legend may be used to group or filter tasks based upon data values.",
    "supportUrl": "https://community.powerbi.com",
    "gitHubUrl": "https://github.com/Microsoft/powerbi-visuals-gantt"
  },
  "apiVersion": "2.6.0",
  "author": {
    "name": "Microsoft",
    "email": "pbicvsupport@microsoft.com"
  },
  "assets": {
    "icon": "assets/icon.png"
  },
  "externalJS": [
  ],
  "style": "style/visual.less",
  "capabilities": "capabilities.json",
  "stringResources": []
}`

My Package.json file

`{
  "name": "visual",
  "private": true,
  "dependencies": {
    "d3": "5.11.0",
    "powerbi-visuals-utils-dataviewutils": "2.2.1",
    "core-js": "3.2.1"
  },
  "devDependencies": {
    "@types/d3": "5.7.2",
    "less": "3.10.3",
    "less-loader": "5.0.0",
    "powerbi-visuals-api": "^2.6.1",
    "style-loader": "0.23.1",
    "tslint": "5.18.0",
    "tslint-microsoft-contrib": "6.2.0",
    "typescript": "3.6.2",
    "webpack": "^4.40.2"
  },
  "scripts": {
    "postinstall": "npm run cert",
    "pbiviz": "pbiviz",
    "start": "pbiviz start",
    "package": "pbiviz package",
    "lint": "tslint -c tslint.json -p tsconfig.json",
    "test": "pbiviz package --resources --no-minify --no-pbiviz",
    "cert": "pbiviz --install-cert"
  }
}
`

And finally outcomes of running `pbiviz start`

![image](https://user-images.githubusercontent.com/36911889/65572816-dc87a680-df86-11e9-9778-534471786b1f.png)

**Comment by helen508light** [comment]
Is your vusual's class similar to "Visual"?

**Comment by Finrod-Amandil** [comment]
Had the exact same outputs as @JeyanthVictory and tried all steps proposed in here, without succes. But I found a solution at last now: Installing the latest version of Node.

Until now I had Node version 8.9.4, which should be sufficient according to the powerbi-visuals-tools repo readme. Updated Node to latest version (12.13.1), re-installed powerbi-visuals-tools@latest (it complained that the engine is incompatible, it wanted node ^10.0.0, but I ignored it) and created a new visual. On first try calling `pbiviz start` an error occurred (error:0909006C:PEM routines:get_name:no start line), just ran the command again and it worked...

The outputs looked different than in the tutorial but it worked and I was able to continue with the next steps of the tutorial (https://docs.microsoft.com/en-us/power-bi/developer/visuals/custom-visual-develop-tutorial)
